src/main/java/de/retest/recheck/ignore/RecheckIgnoreLocator.java

```diff
@@ -0,0 +1,62 @@
+package de.retest.recheck.ignore;
+
+import static de.retest.recheck.RecheckProperties.RETEST_FOLDER_NAME;
+import static de.retest.recheck.configuration.ProjectConfiguration.RECHECK_IGNORE;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.retest.recheck.configuration.ProjectConfigurationUtil;
+import de.retest.recheck.review.GlobalIgnoreApplier;
+import de.retest.recheck.review.counter.NopCounter;
+import de.retest.recheck.review.workers.LoadFilterWorker;
+
+public class RecheckIgnoreLocator {
+
+	private static final Logger logger = LoggerFactory.getLogger( RecheckIgnoreLocator.class );
```

Comment:
I would strive for `@Slf4j` annotations, so that our loggers are the same.

Reply:
Done